### PR TITLE
roundstart snails

### PR DESCRIPTION
**Pull request by dwasint**
# About The Pull Request

## Why It's Good For The Game
👎 saul wants them

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog

:cl:
nerfs:snail crawl speed makes it normal
adds: round start snails
/:cl:

### CODE REVIEW

**Comment by lmenvs** [comment]
cringe

**Comment by Serxule** [comment]
i say keep the crawl speed atleast a little bit faster, but not by a lot, maybe 4 or 4.5, what are the downsides anyways?

**Review by nednaZ — CHANGES_REQUESTED** [review]
Their balance may be a bit of a mess, since anyone can just lay down and start slipping people.

File: code/datums/components/snail_crawl.dm [hunk 1]
```diff
@@ -11,8 +11,8 @@
 	if(snail.resting && !snail.buckled) //s l i d e
 		var/turf/open/OT = get_turf(snail)
 		if(isopenturf(OT))
-			OT.MakeSlippery(TURF_WET_LUBE, 20)
-		snail.add_movespeed_modifier(MOVESPEED_ID_SNAIL_CRAWL, update=TRUE, priority=100, multiplicative_slowdown=-7, movetypes=GROUND)
+			OT.MakeSlippery(TURF_WET_LUBE, min_wet_time = 10, wet_time_to_add =5)
```
[the remark this answers is not in the document]
Comment (dwasint) [reply code/datums/components/snail_crawl.dm:14]: i think its balanced as is they move at half speed, so giving them a lube floor that they can realistically only apply to 2 tiles before they vanish is okay, Plus you can just out walk a snail regardless of crawling or not

**Pull request closed without merging**